### DISCUSSION TITLE
How can I stop the AVNC?

### DISCUSSION THREAD

**Discussion by agl7** [gujjwal00/avnc#188] [General]
Hello gujjwal00,

I don't know how I can stop the AVNC.
How can I stop it?

I have Android 11 on a OnePlus 6 Smartphone.

Best Regards
Andreas (agl7)

**Comment by gujjwal00** [comment]
You should simply press the Back button in navigation bar. Is it not working for you?

**Comment by agl7** [comment]
Hello gujjwal00,

at first, I didn't see any navigation buttons. By chance I noticed that when I swipe up from the bottom edge with my finger, the navigation buttons appear.

Best Regards
Andreas
